### ISSUE TITLE
Documentation is in narrow columns

### ISSUE THREAD

**Issue by gregw** [jetty/jetty.website#32]
The documentation pages do not use wide windows. All the text is in a narrow column with lots of whitespace either side.
On a mobile device, it is kind of strange that the left menu is collapsed, by the the page contents is always listed. Can we make the Contents display collapsed on a small device, as often you only need the contents links to skip the contents that fills the first page.

**Comment by jmcc0nn3ll** [comment]
I can change this back to the original way it was, more like the Spring documentation. I switched it to center for Simone because it was a massive right-side margin for him @mojavelinux said that is typically a DPI setup issue.

**Comment by mojavelinux** [comment]
Numerous studies have showed that the text of an article should be limited to somewhere in the neighborhood of 75 to 85 characters or else it starts to cause eye strain and reader fatige. (Source: https://www.smashingmagazine.com/2014/09/balancing-line-length-font-size-responsive-web-design/#the-ideal-measure-45-to-75-characters). That's why the text is a fixed width (and is on most sites). However, if the scaling settings are not set properly on the computer, this can cause a lot of empty space to appear on the page. A possible solution is to scale the page so the article column uses more of the available width. But it can be difficult to detect when this is necessary and usually requires JavaScript. And if the font is too big, that's also a cause for strain.

My computer has a HiDPI screen with properly configured scaling and this is what I see when I view the pages:

![sample-page](https://github.com/jetty/jetty.website/assets/79351/c9265d74-3f4f-4baa-942c-2749371be352)

If that's not what you see, then your scaling settings are incorrect. I understand that we cannot assume that all visitors have scaling configured correctly (and many don't), but it's not my area of expertise to know what needs to be done to normalize it.

**Comment by jmcc0nn3ll** [comment]
Thanks for adding that information here, Dan; I have reverted the center justify, so it should be back to the default Antora layout in subsequent builds.

**Comment by gregw** [comment]
If readers find more that 85 characters per line causes strain, then they are free to narrow their windows.    That is what windows are for!
If I widen my window, then I'm saying I want more horizontal space, not bigger text.  If I want bigger text I can scroll-zoom.
I don't want bigger text, because then I lose vertical lines and reduce the data I can see.

I should be able to set the default font size of my browser to what I find easy to read and then scale my window to the width I want.    The contents should follow what the reader wants, not make assumptions and end up not using screen real estate.

Currently if I buy a huge 8K display, then this website will show exactly the same amount of data on it as a 14" 1024p display.     I bought a big screen to see more, not to see things bigger!

**Comment by joakime** [comment]
I agree with Greg, this is the web, not printed documentation, I personally find narrow column documentation to be irritating and more difficult to read than normal web layout.

Not even the champion of readability, The New York Times, believes in this kind of adjustment for non-print media.

**Comment by jmcc0nn3ll** [comment]
Regardless of what people want right now, this is what we have. Once all of this lands and we have a working Antora-based site for jetty.org, with all of the Jetty documentation pulled in and themed as one cohesive whole, we can look at finding someone who can take whatever requirements we want and make a custom UI. Perhaps we should have a new Issue that starts to gather custom UI requirements. Features like fully integrated javadoc, fully spaced out text like some of you want, llm backed search, etc.

**Comment by gregw** [comment]
@jmcc0nn3ll I'm amazed that this kind of thing is so difficult to customize?  Typically it should just be a change of a single parameter in a style sheet.

Either way, I think it is a genuine issue and should not be closed.

**Comment by gregw** [comment]
Indeed there is a single property `--doc-max-width--desktop` which when I set to 80% gives me a much better layout.

**Comment by jmcc0nn3ll** [comment]
you can check it out at https://jetty.github.io/jetty.website/index.html

**Comment by gregw** [comment]
> Tested locally, waiting on UI build, and then will update the published site. I don't particularly like it, but you seem adamant so we'll give it a shot.

Thanks.  Yes I am adamant about  this one.   When the primary purpose is to convey the content rather than the style, the user should be in control of such things as page width.   There are definitely situations where the opposite is true and a creator may wish to constrain the style as well, but in this case, if I'm trying to read documentation and want to see more info within my large windows, then the reader should be in control.
